detectors_comparison.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+import cv2
+import numpy as np
+
+video_name = "ch01_20210909192521"
+path_mediapipe = "/Users/diana.kulich/Documents/Masters/dissertation/mediapipe_results"
+path_openpose = "/Users/diana.kulich/Documents/Masters/dissertation/open_pose_results"
+path_yolo = "/Users/diana.kulich/Documents/Masters/dissertation/yolov7/runs/test"
+save_path = "/Users/diana.kulich/Documents/Masters/dissertation/detectors_comparison"
```

Comment:
Shouldn't  it be dynamic paths? Like shouldn't they be provided from comand line?